pep8.py

```diff
@@ -1018,7 +1018,15 @@ def explicit_line_join(logical_line, tokens):
                 parens -= 1
 
 
-def break_around_binary_operator(logical_line, tokens):
+def _is_binary_operator(token_type, text):
+    # The % character is strictly speaking a binary operator, but the
+    # common usage seems to be to put it next to the format parameters,
+    # after a line break.
+    return ((token_type == tokenize.OP or text in ['and', 'or']) and
```

Comment:
Would there be any objections to changing this line (and in the future other similar lines) to something like:

```python
token_op = token_type == tokenize.OP
and_or = text in ['and', 'or']
not_symbol = text not in "()[]{},:.;@=%~"

return ((token_op or and_or) and not_symbol)
```

Not only does this make the line wrapping issue go away, but I think it makes it a bit clearer what is going on. Note: These are just sample variables names, maybe there are even better names for them.